sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -107,12 +107,17 @@ private[hive] object IsolatedClientLoader extends Logging {
         s"Please set ${HiveUtils.HIVE_METASTORE_VERSION.key} with a valid version.")
   }
 
+  def supportHadoopShadedClient(hadoopVersion: String): Boolean = hadoopVersion match {
+    case "3.2.2" => true
```

Comment:
Maintaining a list of versions is probably going to become a pain, is there a semantic version comparison function hanging around somewhere we can use to verify `isMoreRecentVersion(hadoopVersion, "3.2.2")` ?

Reply:
Yup that is a good idea. I'll update it later.